features/integration/simulate.feature

```diff
@@ -182,3 +182,25 @@ Feature: Simulating transactions
     # Final step to check extra budgets in simulation result
     Then I check the simulation result has power packs extra-opcode-budget with extra budget 2000.
     And the simulation should succeed without any failure message
+
+  @simulate.exec_trace_with_stack_scratch
```

Comment:
Probably should add this to the integration tests tag table in the README: https://github.com/algorand/algorand-sdk-testing#integration-tests

Reply:
ugh ok... actually we probably want to add all the previous sub-simulate tags under readme tho.

Reply:
see [readme table](https://github.com/algorand/algorand-sdk-testing/pull/301/commits/d59df80f82de0d9d00c1d7fa1e291d27b2c79a6c)